tests/e2e/flowtest/test_scc.py

```diff
@@ -0,0 +1,55 @@
+import pytest
```

Comment:
This may not be a relevant place for this test, since the scc is applied at cluster level should this be placed it under z_cluster?

Reply:
ack

Reply:
@PrasadDesala @hnallurv we may need a different way to handle these new tests. one option is what Pratik suggested. Another option is that we could have someone from the e2e team join the brown squad and help with the analysis and spread the knowledge within the squad.